pkg/graph/resolvers/system_intake_grb_discussions.go

```diff
@@ -0,0 +1,80 @@
+package resolvers
+
+import (
+	"context"
+	"errors"
+
+	"github.com/jmoiron/sqlx"
+
+	"github.com/cms-enterprise/easi-app/pkg/appcontext"
+	"github.com/cms-enterprise/easi-app/pkg/email"
+	"github.com/cms-enterprise/easi-app/pkg/models"
+	"github.com/cms-enterprise/easi-app/pkg/services"
+	"github.com/cms-enterprise/easi-app/pkg/sqlutils"
+	"github.com/cms-enterprise/easi-app/pkg/storage"
+)
+
+// CreateSystemIntakeGRBDiscussionPost creates an initial GRB discussion post
+func CreateSystemIntakeGRBDiscussionPost(
+	ctx context.Context,
+	store *storage.Store,
+	emailClient *email.Client,
+	input models.CreateSystemIntakeGRBDiscussionPostInput,
+) (*models.SystemIntakeGRBReviewDiscussionPost, error) {
+	return sqlutils.WithTransactionRet(ctx, store, func(tx *sqlx.Tx) (*models.SystemIntakeGRBReviewDiscussionPost, error) {
```

Comment:
it appears there is only one use of the `tx` in the transaction, can this be unwrapped?

Reply:
I'd be fine if we dropped this just for clarity, but it also shouldn't _really_ cause any issues as-is

Though, notably, since we're calling a dataloader in this function, that will _not_ actually be part of this transaction, so it could be argued that it's a little misleading since there's only the one function call that actually uses it.

Reply:
Though, if we end up making more storage calls to fetch stuff for emails, we might end up wanting it. If that's the case we can keep it!

Reply:
good call! considering we may store tags one day, i like that